openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -135,4 +139,9 @@ def count_data():
             'students_per_country': json.dumps(students_per_country)
         })
 
-    requests.post(post_url, data)
+    try:
+        request = requests.post(post_url, data)
+        logger.info('Connected without error to {0}'.format(request.url))
+        logger.info('Data was successfully sent. Status code is {0}.'.format(request.status_code))
+    except requests.ConnectionError as error:
```

Comment:
@TamoshaytisV, log in terminal shows me `ConnectionError` exception few times, what is reason to follow `RequestException`?